pkg/addons/data/Deployment.apps/openshift-monitoring/cluster-monitoring-operator.yaml

```diff
@@ -14,15 +14,14 @@ spec:
     rollingUpdate:
       maxSurge: 1
```

Comment:
I am wondering what does it means to be running two different versions of the cluster monitoring operator at the same time.